pkgs/top-level/all-packages.nix

```diff
@@ -20246,9 +20246,7 @@ with pkgs;
     go = buildPackages.go_1_16;
   };
   # go_1_17 has go module changes which may not be portable
-  # across different go versions and/or platforms,
-  # it also requires >=10.13 stdenv on darwin which
-  # is not currently available for x86_64-darwin
+  # across different go versions and/or platforms
```

Comment:
A nixpkgs issue link here would be great, otherwise it's not clear what exactly needs to happen.

Reply:
Good idea. I don't think there is an issue for this, but I think this comment sums up the issue: https://github.com/NixOS/nixpkgs/pull/127519#issuecomment-864726365

Sound good?

Reply:
I'm on the road so here is a placeholder issue: https://github.com/NixOS/nixpkgs/issues/144667